cypress/integration/common/priority_queue.spec.js

```diff
@@ -120,7 +120,7 @@ describe("TA Priority Queue", () => {
   });
 
   describe("Requeue Student", () => {
-    it.only("Requeue student works", function () {
+    it("Requeue student works", function () {
```

Comment:
nice find! I wonder if there is a way to have the linter catch this in the future?

Reply:
same